tutorials/ctscan.py

```diff
@@ -44,9 +44,11 @@ def radoncurve(x, r, theta):
     )
 
 
-x = np.load("../testdata/optimization/shepp_logan_phantom.npy")
+x = np.load(
+    "/Users/ravasim/Desktop/KAUST/OpenSource/pylops/testdata/optimization/shepp_logan_phantom.npy"
+).T
```

Comment:
@mrava87 Could you revert the path change?

Reply:
Sorry, I was checking this in a notebook and forgot to change it back. It is now fixed (made a PR straight after...)

Reply:
https://github.com/PyLops/pylops/commit/801e51b2da89245abd743c360153051342b358bb